packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -366,10 +366,14 @@ export const prepareVaultDirector = (
           const factoryPowers = Far('vault factory powers', {
             getGovernedParams: () =>
               Far('vault manager param manager', {
+                // one param from director scope
```

Comment:
I don't understand the significance of this, is there a hazard here?

Reply:
Not a runtime hazard but the Far says it's a "vault manager param manager" and while it has all the "vault manager" params it has exactly one param from the director scope.